.eslintrc.json

```diff
@@ -10,30 +10,11 @@
     "commonjs": true,
     "es6": true
   },
+  "root": true,
   "rules": {
-    "indent": [
-      "error",
-      2,
-      {
-        "SwitchCase": 1,
-        "VariableDeclarator": {
-          "var": 2,
-          "let": 2,
-          "const": 3
-        }
-      }
-    ],
-    "linebreak-style": [
-      "error",
-      "unix"
-    ],
-    "quotes": [
-      "error",
-      "double"
-    ],
-    "semi": [
-      "error",
-      "always"
-    ]
+    "indent": ["error", 2, {"SwitchCase": 1, "VariableDeclarator": {"var": 2, "let": 2, "const": 3}}],
+    "linebreak-style": ["error", "unix"],
+    "quotes": ["error", "double"],
+    "semi": ["error", "always"]
```

Comment:
Just flattened all these to make them easier to sort+scan.

Reply:
Since you have a GitHub org, not sure if you want to do a custom **eslint-config-lockbox** and/or **eslint-plugin-lockbox** so you can share ESLint configs between repos for cross repo consistency. Or whatever.

Reply:
we probably do.  But I'm not holding up things to wait for that (-: